src/odemis/gui/util/img.py

```diff
@@ -2621,37 +2621,38 @@ def insert_tile_to_image(tile, ovv):
     return ovv
 
 
-def merge_screen(im, background):
+def merge_screen(ima, imb):
     """ 
-    Merges two images (im and background) into one using the "screen" operator:
+    Merges two images into one using the "screen" operator:
```

Comment:
Maybe you could add in the first sentence how they are merged. Before it seemed that _im_ would be infront of _background_.
Now maybe now the first sentence can be changed to something like:
> Merges two images by averiging between the values into one using the "screen" operator: f(xA,xB) = xA + xB − xA·xB (with values between 0 and 1, with each channel independent)

I'm never used a "screen" operator but since you are clarifying the doc string it would be nice if you can directly have an intuitive idea of how the function works if you the docstring (without going into maths etc.)

Reply:
I don't have a very good "intuitive description. "Averaging" would be wrong. After searching online, I saw 
"soft plus", which is the less bad, I've found. I'll update, unless you have a better suggestion.